src/actinia_core/core/stac_importer_interface.py

```diff
@@ -95,37 +93,36 @@ def _apply_filter(stac_root_search, stac_name, interval, bbox, filter):
 
         full_filtered_result = stac_search.json()
 
-        if "features" in full_filtered_result:
+        if "features" in full_filtered_result\
+                and len(full_filtered_result["features"]) > 0:
             return full_filtered_result
         else:
-            raise AsyncProcessError(full_filtered_result)
+            raise AsyncProcessError("Not matched found")
 
     @staticmethod
     def _get_filtered_bands(stac_items, semantic_label):
         band_roots = {}
 
         for feature in stac_items["features"]:
+            item_date = feature["properties"]["datetime"]
             for key, value in feature["assets"].items():
                 if "eo:bands" in value:
                     if "common_name" in value["eo:bands"][0]:
-                        if value["eo:bands"][0]["common_name"] in semantic_label:
-                            band_name = value["eo:bands"][0]["name"]
-                            if band_name not in band_roots:
-                                band_roots[band_name] = {}
-                            feature_id = feature["id"]
-                            item_link = feature["assets"][band_name]["href"]
-                            band_roots[band_name][feature_id] = item_link
-                        elif value["eo:bands"][0]["name"] in semantic_label:
+                        if value["eo:bands"][0]["common_name"] in semantic_label\
+                                or value["eo:bands"][0]["name"] in semantic_label\
```

Comment:
whitespace before `\`, check with black or flake8
```suggestion
                        if value["eo:bands"][0]["common_name"] in semantic_label \
                                or value["eo:bands"][0]["name"] in semantic_label \
```